tests/test_apiclient.py

```diff
@@ -363,6 +363,15 @@ def test_search_services_with_pagination(self, search_client, rmock):
         assert result == {'services': "myresponse"}
         assert rmock.last_request.query == 'page=10'
 
+    def test_search_services_id_only(self, search_client, rmock):
+        rmock.get(
+            'http://baseurl/g-cloud/services/search?idOnly=True',
+            json={'services': "myresponse"},
+            status_code=200)
+        result = search_client.search_services(index='g-cloud', id_only=True)
+        assert result == {'services': "myresponse"}
+        assert rmock.last_request.query == 'idonly=true'
```

Comment:
I suppose this must be case-insensitive, but seems weird to me that the "o" is not capitalized.

Reply:
It's case sensitive, but weirdly makes it lowercase...